src/app/containers/PageHandlers/__snapshots__/withError.test.jsx.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`withError HOC with error should return the errorMain component 1`] = `
+<ErrorMain
+  status={500}
+/>
+`;
+
+exports[`withError HOC with no error should return the passed in component 1`] = `
```

Comment:
Is this the expected output for this test?

Reply:
Good spot, I guess technically it is as I am not passing the correct props in the test :D Updating now.